### DISCUSSION TITLE
Does the lack of await on other assertion libraries like Shouldly affect the tests negatively?

### DISCUSSION THREAD

**Discussion by macfarmw** [thomhurst/TUnit#935] [Q&A] [answered]
I'm trying out some tests with Shouldly assertions. I defined the test as `public void` instead of `public async Task` and used the Shouldly assertion as normal. The tests appear to be working fine this way. Is there any negative impact that this usage will have on the tests that might not be obvious?

**Comment by thomhurst** [comment, marked as answer]
Nope that's fine! The `await` in TUnit assertions is what actually 'invokes/triggers' the assertion to evaluate.

Shouldly doesn't work like that, and that's fine, so no await or async is necessary 😄
